src/containers/ListingPage/ListingPage.js

```diff
@@ -59,6 +59,10 @@ import SectionRulesMaybe from './SectionRulesMaybe';
 import SectionMapMaybe from './SectionMapMaybe';
 import css from './ListingPage.css';
 
+import { storeData } from '../CheckoutPage/CheckoutPageSessionHelpers';
+// This storage key is used in sessionStorage and it needs to be same as on CheckoutPage
+const STORAGE_KEY = 'CheckoutPage';
+
```

Comment:
We need to call `storeData` function already on ListingPage so that the user can be redirected directly to CheckoutPage after authenticating with idp (Facebook). 

Maybe we should move the session helpers file from CheckoutPage to utils and change the storage key to something else now that this is not restricted just to CheckoutPage. But I wanted to know if this setup makes sense before doing this change.

Reply:
@OtterleyW Why can't you use "callSetInitialValues(setInitialValues, initialValues);"?